jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaHandlerResolver.kt

```diff
@@ -202,6 +202,7 @@ class JavaLambdaHandlerResolver : LambdaHandlerResolver {
 
     private fun PsiParameter.isClass(classFullName: String): Boolean {
         val className = (this.type as? PsiClassReferenceType)?.className ?: return false
```

Comment:
This method is a rather round about way of figuring this out. Just have IntelliJ compare the types.

```kotlin
    private fun PsiParameter.isClass(classFullName: String): Boolean {
        return this.type.isAssignableFrom(
            PsiType.getTypeByName(
                classFullName,  
                project,
                GlobalSearchScope.projectScope(project)
            )
        )
    }
```

Reply:
This is neat. Updated.